R/convert_output.R

```diff
@@ -1,15 +1,11 @@
 #' Convert Output
 #'
 #' Format stock assessment output files to a standardized format.
-#'
+
+#' @inheritParams create_template
```

Comment:
In most of these functions, the arguments will be changes from the other PR so we don't really have to make many changes. This could cause issues when merging, but we'll figure it out

Reply:
👍 True. I'll keep this as-is, then, and fix it later.